Pomodoro.Services/Pomodoro - Backup.Services.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Pomodoro.Core\Pomodoro.Core.csproj" />
+    <ProjectReference Include="..\Pomodoro.DataAccess\Pomodoro.DataAccess.csproj" />
+  </ItemGroup>
+
+</Project>
```

Comment:
What's the purpose of this file?

Reply:
i'm sorry, my IDE automatically added this file and i didn't see this